src/types/FilterType.tsx

```diff
@@ -0,0 +1,5 @@
+export enum FilterType {
```

Comment:
you can create an enum for errors as well.

Reply:
done) it really looks better that way))